Add support for tuple type in the Cassandra connector (#25516)
prestodb/presto: presto-docs/src/main/sphinx/connector/cassandra.rst

Any collection (LIST/MAP/SET) can be designated as FROZEN, and the value is
mapped to VARCHAR. Additionally, blobs have the limitation that they cannot be empty.

> Types not mentioned in the table above are not supported (e.g. UDT).

Reviewer: ```suggestion
Types not mentioned in the table above, such as UDT, are not supported.
```
Avoid Latin phrases and abbreviations in documentation. See the [GitLab documentation style guide entry for e.g.](https://docs.gitlab.com/development/documentation/styleguide/word_list/#eg) for alternatives. 

I revised the sentence as one of several ways to fix this, but if you find one you like better please use it instead.

Author: It is phrased a little strange, I'll fix it up.
